x-pack/plugins/ml/public/application/trained_models/models_management/test_models/models/text_classification/fill_mask_inference.ts

```diff
@@ -17,6 +17,14 @@ const MASK = '[MASK]';
 
 export class FillMaskInference extends InferenceBase<TextClassificationResponse> {
   protected inferenceType = SUPPORTED_PYTORCH_TASKS.FILL_MASK;
+  protected info = [
```

Comment:
Contents of the callout can be supplied by adding an array of sentences in the inference class.

Reply:
Thanks!  I've copied the short sentences from https://github.com/elastic/kibana/pull/134795 into this format via https://github.com/jgowdyelastic/kibana/pull/1. Let me know if I can go ahead and merge into this PR

Reply:
Thanks!  I cherry-picked fb6f98ac07ca41601231075278df61a3432e266f